development/tools/cmd/githubstats/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/jamiealquiza/envy"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2"
+)
+
+type Issues struct {
+	Open   IssuesOpen
+	Closed IssuesClosed
+}
+
+type IssuesOpen struct {
```

Comment:
why it's duplicated?

why u cannot have just 

```go
type Issues struct {
	Open   IssuesStats
	Closed IssuesStats
}
```

Reply:
Right, I somehow forgot about that